### ISSUE TITLE
wiki: add a line continuation symbol for tables

### ISSUE THREAD

**Issue by vadosnaprimer** [TASVideos/tasvideos#1656]
Some tables are very hard to edit when a row has to be represented by an actual line, and there are links that we have to include, or many columns.

Examples:
https://tasvideos.org/Wiki/ViewSource?path=EmulatorResources
https://tasvideos.org/Wiki/ViewSource?path=EmulatorResources%2FNESAccuracyTests
https://tasvideos.org/Wiki/ViewSource?path=FastestCompletion

**Comment by nattthebear** [comment]
I appreciate the problem, but is this a good solution?  The wiki syntax we inherited is an absolute mess, with no universal escaping, opaque escaping rules, and complex, hard to follow parsing rules.  The codebase implementation isn't that bad, but it can't be any less complex than the language it's representing.

This would be adding another special rule, with backwards incompatibility and no escaping possible, that would have to be just bolted onto the existing parser.
